graphql/src/schema/meta.graphql

```diff
@@ -1,6 +1,28 @@
+# GraphQL core functionality
+scalar Boolean
+scalar ID
+scalar Int
+scalar Float
```

Comment:
We don't have a `Float` type - not sure what will happen if somebody tries to use that, but I'd suspect that it will blow up in some way.

Reply:
Thanks for the review @lutter ! 
We are using it as part of the introspection schema (https://github.com/graphprotocol/graph-node/blob/393d0dca8c25a184f140da7cc62596a26d4580e5/graphql/src/introspection/schema.rs#L17), and it's also a built-in in GraphQL, so are you sure it's a good direction to drop it?

Reply:
I rebased and pushed a fix for it, but not merging yet, waiting for your decision on that. 
Thanks!

Reply:
I just checked the code, and we actually do handle it - you are right, we should leave the `Float` in then.

Reply:
Fixed, thanks! will merge when CI clears